db/post_migrate/20210922064213_alter_bumped_at_indexes_on_topics.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class AlterBumpedAtIndexesOnTopics < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def up
+    execute(<<~SQL)
+    CREATE INDEX CONCURRENTLY idx_public_topics_front_page
```

Comment:
Nitpick: do you know if we have a convention on naming indexes? I see that in `topic.rb` most of them start with `index_topics_on_...`.

Reply:
We've never had a convention for index name. Those that start with `index_topics_on...` are usually generated by ActiveRecord while the `idx_` prefix or rather a custom index name is used when the index name generated by ActiveRecord is too long